docs/topics/classes.md

```diff
@@ -71,7 +71,7 @@ class Customer(name: String) {
 In fact, for declaring properties and initializing them from the primary constructor, Kotlin has a concise syntax:
 
 ```kotlin
-class Person(val firstName: String, val lastName: String, var age: Int) { /*...*/ }
+class Person(val firstName: String = "Foo", val lastName: String = "Bar", var age: Int = 27) { /*...*/ }
```

Comment:
This is a good addition. Thanks.
However, I'd suggest adding another sentence and this example after the current one, not instead of it. 
This will separate two steps: _declaring properties as constructor parameters_ and _setting their default values_.
What do you think?

One more thing: please use a real name, not `Foo/Bar`.